viadot/sources/vid_club.py

```diff
@@ -80,70 +81,134 @@ def build_query(
             )
         return url
 
+    def intervals(
+        self, from_date: str, to_date: str, days_interval: int
+    ) -> Tuple[List[str], List[str]]:
+        """
+        Breaks dates range into smaller by provided days interval.
+
+        Args:
+            from_date (str): Start date for the query.
+            to_date (str): End date for the query. By default, datetime.today() will be used.
+            days_interval (int): Days specified in date range per api call (test showed that 30-40 is optimal for performance).
+
+        Returns:
+            List[str], List[str]: Starts abd Ends lists that contains information about date ranges for specific period and time interval.
+        """
+
+        if to_date == None:
+            end_date = datetime.today().date()
+        else:
+            end_date = datetime.strptime(to_date, "%Y-%m-%d").date()
+        start_date = datetime.strptime(from_date, "%Y-%m-%d").date()
```

Comment:
Please add empty line between line 102 and 103 for better clarity

Reply:
done